ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -183,24 +181,36 @@ TFLiteInterpreter::invoke (const GstTensorMemory * input,
   gint64 start_time = g_get_real_time ();
 #endif
 
+  std::vector <int> tensors_idx;
+  int tensor_idx;
+  TfLiteTensor *tensor_ptr;
   TfLiteStatus status;
   bool size_mismatch = FALSE;
 
-  for (unsigned int i = 0; i < input_tensor_ptr.size (); i++) {
-    if (input_tensor_ptr[i]->bytes != input[i].size)
+  for (unsigned int i = 0; i < outputTensorMeta.num_tensors; ++i) {
+    tensor_idx = interpreter->outputs ()[i];
+    tensor_ptr = interpreter->tensor (tensor_idx);
+
+    if (tensor_ptr->bytes != output[i].size)
       size_mismatch = TRUE;
-    input_tensor_ptr[i]->data.raw = (char *) input[i].data;
+    tensor_ptr->data.raw = (char *) output[i].data;
+    tensors_idx.push_back (tensor_idx);
   }
-  for (unsigned int i = 0; i < output_tensor_ptr.size (); i++) {
-    if (output_tensor_ptr[i]->bytes != output[i].size)
+
+  for (unsigned int i = 0; i < inputTensorMeta.num_tensors; ++i) {
+    tensor_idx = interpreter->inputs ()[i];
+    tensor_ptr = interpreter->tensor (tensor_idx);
+
+    if (tensor_ptr->bytes != input[i].size)
       size_mismatch = TRUE;
-    output_tensor_ptr[i]->data.raw = (char *) output[i].data;
+    tensor_ptr->data.raw = (char *) input[i].data;
+    tensors_idx.push_back (tensor_idx);
   }
 
   if (!size_mismatch) {
 #ifdef ENABLE_TFLITE_NNAPI_DELEGATE
     if (use_nnapi)
-      status = nnfw_delegate->Invoke (interpreter.get ());
+      status = nnfw_delegate->Invoke (interpreter.get());
```

Comment:
this style change is not the recommended way.

Reply:
Thank you for pointing out. It's fixed.